ophyd/scaler.py

```diff
@@ -160,5 +161,3 @@ def select_channels(self, chan_names):
                                    "{}".format(ch, tuple(name_map)))
         self.channels.read_attrs = list(read_attrs)
         self.channels.configuration_attrs = list(read_attrs)
-        self.hints = {'fields': [getattr(self.channels, ch).s.name
```

Comment:
This is even trickier than the case above. I think that instead of delete this, it's probably easier to change `self.hints` -> `self._hints` and treat this as if it's a user-provided value.